packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const chalk = require('chalk');
+const dotenv = require('dotenv');
+const {JSDOM} = require('jsdom');
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
+const domNodesToCdt = require('@applitools/visual-grid-client/src/browser-util/domNodesToCdt');
+const {TestFailedError, TestResults} = require('@applitools/eyes.sdk.core');
+
+dotenv.config();
+
+function getGridClientConfig() {
+  const projectName = 'kieran-grid-test';
+  const branchName = projectName + '/master';
+
+  return {
+    appName: projectName,
+    branchName,
+    batchName: projectName,
+    browser: {
+      name: 'chrome',
+      width: 400,
+      height: 300
+    }
+  };
+}
+
+function getStaticResources(cssFilenames: string[], resourceDir: string) {
+  const resources: any = {};
+  for (const cssFilename of cssFilenames) {
+    const url = '/' + cssFilename;
+    resources[url] = {
+      url,
+      type: 'text/css',
+      value: fs.readFileSync(path.join(resourceDir, cssFilename))
+    };
+  }
+  return resources;
+}
+
+function getStatusIcon(result: any) {
+  return (
+    result.status === 'unmodified' ? chalk.green('=') :
+    result.status === 'new' ? chalk.green('+') :
+    result.status === 'modified' ? chalk.red('≠') :
+    chalk.red('!')
+  );
+}
+
+// We cannot allow any of the result promises to reject, because we're going to
+// await on a bunch of them in parallel using Promise.all, and a single rejected
+// promise would reject the entire batch.
+function getTestResult(testResultPromise: any) {
+  return (
+    testResultPromise
+    .catch((err: any) => err)
+    .then((res: any) => res instanceof TestFailedError ? res.getTestResults() : res)
+    .then((res: any) => Array.isArray(res) && res[0] instanceof TestResults ? res[0] : res)
+    .then((res: any) => {
+      if (res instanceof TestResults) {
+        if (res.getIsDifferent()) {
+          return {status: 'modified', url: res.getUrl()};
+        }
+        if (res.getIsNew()) {
+          return {status: 'new'};
+        }
+        if (res.isPassed()) {
+          return {status: 'unmodified'};
+        }
+      }
+
+      return {status: 'error', error: res};
+    })
+  );
+}
+
+async function runTest(gridClient: any, gridClientConfig: any, testName: any, html: any, resources: any) {
+  const dom = new JSDOM(html).window.document;
+
+  const {checkWindow, close} = await gridClient.openEyes({
+    ...gridClientConfig,
+    testName,
+  });
+
+  await checkWindow({
+    url: 'http://localhost/html/index.html',
+    cdt: domNodesToCdt(dom),
+    sizeMode: 'viewport',
+    resourceContents: resources
+  });
+
+  return close();
+}
+
+export async function runEyes() {
+  const resourceDir   = path.join(process.cwd(), 'dist');
+  const cssFilenames  = glob.sync('*.css', {cwd: resourceDir});
```

Comment:
This grabs all CSS files from `dist`, but what if there are some other files that weren't created by our tool? I think we shouldn't even put our CSS and HTML files into dist, because we don't plan to publish them to npm. Ideally we should use a temp folder. There's a `tmp` package on npm that does this.